crates/node/core/src/args/rpc_server.rs

```diff
@@ -156,6 +156,12 @@ pub struct RpcServerArgs {
     )]
     pub rpc_gas_cap: u64,
 
+    /// The maximum proof window for historical proof generation.
+    /// This value allows for generating historical proofs up to
+    /// configured number of blocks from current tip (up to `tip - window`).
+    #[arg(long = "rpc.eth-proof-window", default_value_t = constants::DEFAULT_ETH_PROOF_WINDOW)]
```

Comment:
should we enforce some restrictions here, via a rangevalueparser? otherwise this can be configured with u64::Max ?